Sources/FluentKit/Properties/Children.swift

```diff
@@ -1,26 +1,20 @@
 @propertyWrapper
 public final class Children<From, To>: AnyProperty
-    where From: Model, To: Model
+    where From: ModelIdentifiable & Codable & CustomStringConvertible, To: Model
```

Comment:
I'm worried about the implications that losing strong type information on `Children.From` / `Parent.To` could have on future additions to these types. You're getting around it well by using the `String` key instead of `Model.IDValue` and other smart tricks, but these workarounds will have to grow more complex as Fluent's features do. Having the fully qualified model types in these relations (as `@OptionalParent` allows) means Fluent has unfettered access to all of the APIs `Model` offers, like accessing internal storage and fields, without needing to type-erase or add additional protocols / requirements.